gallery/gallery-library/src/integration-test/java/org/silverpeas/components/gallery/stub/SessionManagementStub.java

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (C) 2000 - 2021 Silverpeas
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * As a special exception to the terms and conditions of version 3.0 of
+ * the GPL, you may redistribute this Program in connection with Free/Lib
+ * Open Source Software ("FLOSS") applications as described in Silverpeas
+ * FLOSS exception. You should have received a copy of the text describin
+ * the FLOSS exception, and it is also available here:
+ * "https://www.silverpeas.org/legal/floss_exception.html"
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public Licen
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package org.silverpeas.components.gallery.stub;
+
+import org.silverpeas.core.admin.user.model.User;
+import org.silverpeas.core.admin.user.model.UserDetail;
+import org.silverpeas.core.annotation.Service;
+import org.silverpeas.core.security.session.SessionInfo;
+import org.silverpeas.core.security.session.SessionManagement;
+import org.silverpeas.core.security.session.SessionValidationContext;
+
+import javax.enterprise.inject.Alternative;
+import javax.inject.Singleton;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpSession;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.UUID;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
+
+@Service
+@Singleton
+public class SessionManagementStub implements SessionManagement {
```

Comment:
I do not understand this implementation of this SessionManagementStub against the one implemented into core-web-test?

Reply:
The pb is that the CDI container doesn't find the implementation in the core-web-test and then fails to deploy the test archive through Arquillian. I don't know why it doesn't find it (as there is a _beans.xml_ descriptor in the core-web-test jarlib)

Reply:
:thumbsup: